simpleclient/src/main/java/io/prometheus/client/exemplars/Exemplar.java

```diff
@@ -0,0 +1,188 @@
+package io.prometheus.client.exemplars;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+/**
+ * Immutable data class holding an Exemplar.
+ */
+public class Exemplar {
+
+  public static final String TRACE_ID = "trace_id";
+  public static final String SPAN_ID = "span_id";
+
+  private final String[] labels;
+  private final double value;
+  private final Long timestampMs;
+
+  private static final Pattern labelNameRegex = Pattern.compile("[a-zA-Z_]\\w*"); // \w is [a-zA-Z_0-9]
```

Comment:
This comment only confuses matters, if you want to say `[a-zA-Z_0-9]` then say `[a-zA-Z_0-9]` directly.

Reply:
true

Reply:
> Does that address your concerns?

No, the problem was never maven. It was some other Java build tool that was reading the poms.